mocha/common.js

```diff
@@ -0,0 +1,8 @@
+global.React = require('react');
```

Comment:
I don't think it's just you. We pulled this stuff over from other projects. Let's get rid of this stuff and import explicitly as needed.

> "It's the more proper thing to do"
– Marinus, 2016